src/utils/random.js

```diff
@@ -0,0 +1,11 @@
+// TODO remove this module
```

Comment:
Can't we use sampler.js?

Reply:
discussed offline, sampler.js will be removed